templates/common/__init__.py.in

```diff
@@ -183,22 +183,29 @@ def Analyze(command, sa_clang_compiler, sa_formatting=False, sa_modernize="", sa
     Perform static analysis and check of formatting rules if requested
     '''
     returnVal=0
+    formatting_on = os.environ.has_key("SA_FORMATTING") or sa_formatting
```

Comment:
`has_key` is deprecated (removed in Python 3), can you switch to `"..." in os.environ`